### PR TITLE
Changed table filtering system to new variant

### PR DESCRIPTION
**Pull request by shaneYeti**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 ### Changes:
 
+-   Improvements
+    -   Changed how the file filtering functions in the table view.
```
Comment (KallynGowdy) [inline CHANGELOG.md]: @shaneYeti Is this a typo?

Comment (shaneYeti) [reply CHANGELOG.md]: @KallynGowdy Which part of the statement?

Comment (KallynGowdy) [reply CHANGELOG.md]: @shaneYeti Well, it doesn't seem to be a complete statement.

**Comment by shaneYeti** [comment]
@KallynGowdy  This any better?

**Pull request merged**